docs/main.md

```diff
@@ -0,0 +1,157 @@
+# Medical  <img src="_static/cat-logo.svg" width=20>oncept Annotation Tool
```

Comment:
Looks the content here is identical to the README.md. Is that possible to include it here and stay DRY?